### PR TITLE
Processing 13c spectrum test case

### PR DESCRIPTION
**Pull request by wadjih-bencheikh18**
closes : https://github.com/cheminfo/nmrium/issues/1692

### CODE REVIEW

File: test-e2e/panels/filters.test.ts [hunk 1]
```diff
@@ -79,3 +84,32 @@ test('process 1d FID 13c spectrum', async ({ page }) => {
     await baselineCorrectionFilter(nmrium);
   });
 });
+
+test('Processing 13c spectrum', async ({ page }) => {
```
Comment (lpatiny) [inline test-e2e/panels/filters.test.ts:88]: You don't need to create a new test but just append the new steps to the previous step.

File: test-e2e/panels/filters.test.ts [hunk 2]
```diff
@@ -57,7 +57,12 @@ async function baselineCorrectionFilter(nmrium: NmriumPage) {
     ),
   ).toBeVisible();
 }
-
+async function xScaleDomain(nmrium: NmriumPage, min: number, max: number) {
```
Comment (lpatiny) [inline test-e2e/panels/filters.test.ts:60]: Please add a method in NmriumPage called assertXScaleDomain(min, max).

We will often reuse it ! And don't hesiate to add new method in NmriumPage

File: test-e2e/panels/filters.test.ts [hunk 3]
```diff
@@ -57,7 +57,12 @@ async function baselineCorrectionFilter(nmrium: NmriumPage) {
...
+  const firstTick = nmrium.page.locator('.x >> .tick >> nth=0');
```
Comment (lpatiny) [inline test-e2e/panels/filters.test.ts:61]: You could try to refactor to use something like:
```js
const xTicks = page.locator(‘.x >> .tick’)
const firstTick = xTicks.first()
const lastTick = xTicks.last()
```

File: test-e2e/panels/filters.test.ts (hunk 1 above)
Comment (wadjih-bencheikh18) [reply test-e2e/panels/filters.test.ts:88]: but in the previous test there is more filter used , can i remove them?

Comment (lpatiny) [reply test-e2e/panels/filters.test.ts:83]: You may leave them if there are more filters.

Comment (wadjih-bencheikh18) [reply test-e2e/panels/filters.test.ts:83]: okey thank you

**Pull request merged**